server/leafnode.go

```diff
@@ -649,6 +649,14 @@ func (s *Server) createLeafNode(conn net.Conn, remote *leafNodeCfg) *client {
 			return nil
 		}
 
+		// Not sure that can happen, but in case the connection was marked
+		// as closed during In case connection was marked as closed during parse...
```

Comment:
In -> in

Reply:
whole comment is wrong :-) will fix that.